daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -98,6 +118,30 @@ func rotate(name string, maxFiles int) error {
 	return nil
 }
 
+func copyToFile(outfile string, r io.Reader) error {
```

Comment:
@AkihiroSuda  The data source is an `io.Reader` interface type, so I think using `io.Copy` might be a bit more appropriate here.